Makefile

```diff
@@ -1,6 +1,6 @@
 TESTIMONY_TOKENS="caseautomation, casecomponent, caseimportance, caselevel, caseposneg, description, expectedresults, id, requirement, setup, subtype1, subtype2, steps, teardown, testtype, upstream, title"
 TESTIMONY_MINIMUM_TOKENS="id, requirement, caseautomation, caselevel, casecomponent, testtype, caseimportance, upstream"
-PYTEST_OPTIONS=--verbose
+PYTEST_OPTIONS=--verbose --driver=Chrome
```

Comment:
Maybe you want to add this on the `test-qpc-ui` instead of here. Or not even define it. Because when calling make we can override this by running like `make <target> PYTEST_OPTIONS="--my-option --my-second-option"

Reply:
And this falls again on the issue of not having UI machinery defined and calling any of the make. Since UI machinery is optional and the UI tests will be skipped properly I think passing the driver should be done by the person running the tests.